crates/solvers/src/tests/baseline/bal_liquidity.rs

```diff
@@ -65,12 +65,12 @@ async fn weighted() {
                     "tokens": {
                         "0x6810e776880c02933d47db1b9fc05908e5386b96": {
                             "balance": "11260752191375725565253",
-                            "scalingFactor": "1000000000000000000",
+                            "scalingFactor": "1",
```

Comment:
We were using the incorrect scaling factors in the E2E tests, since they were both equal, it didn't actually affect the output of the functions.